torchbox/wagtail-torchbox
# Adds related course pages

:warning: based on https://github.com/torchbox/wagtail-torchbox/pull/562

### Description of Changes Made

This PR adds the related course page model, so courses can be related to one another.

### How to Test

1. Check out this branch locally
2. Add a course landing page
3. Add 3 course pages
4. Relate some course pages to eachother
5. View front end of course page and you should see the related course pages (unstyled)

### MR Checklist
